packages/components/Menu/src/Menu/Menu.types.ts

```diff
@@ -15,6 +15,7 @@ export interface MenuProps extends MenuListProps {
 
   /**
    * How much delay to have between hover in and showing the menu, in ms.
+   * @platform win32, macOS
```

Comment:
nit: I think I've seen platforms to be all lowercase not not comma separated, like:

```@platform win32 macos```

Reply:
<img width="522" alt="image" src="https://user-images.githubusercontent.com/30728574/232665786-3a53e067-4bfd-46be-929d-05b2db6017a1.png">


@Saadnajmi , i think across repo for platform status we have used same, so went with the same convention.

Reply:
Oh, thanks! I guess I was wrong =)

Reply:
resolving this as it is expected.